n_helpers.c

```diff
@@ -84,205 +84,89 @@ NOTE_C_STATIC void setTime(JTIME seconds);
 NOTE_C_STATIC bool timerExpiredSecs(uint32_t *timer, uint32_t periodSecs);
 NOTE_C_STATIC int ytodays(int year);
 
-static const char BINARY_EOP = '\n';
+static const char NOTE_C_BINARY_EOP = '\n';
 
 //**************************************************************************/
 /*!
-  @brief  Get the length of the data stored on the Notecard. If there's no data
-          stored on the Notecard, then `*len` will return 0.
+  @brief  Decode binary data received from the Notecard.
 
-  @param  len [out] the length of the decoded contents of the Notecard's binary
-          data store.
+  @param  encData The encoded binary data to decode.
+  @param  encLen The length of the encoded binary data.
```

Comment:
Should we change `encLen` to `encDataLen`, to match how we use `decData` and `decDataLen` for `NoteBinaryCodecEncode`?